Sources/TuistDependencies/SwiftPackageManager/SwiftPackageManagerInteractor.swift

```diff
@@ -0,0 +1,190 @@
+import TSCBasic
+import TuistGraph
+import TuistSupport
+
+enum SwiftPackageManagerInteractorError: FatalError, Equatable {
+    /// Thrown when `Package.resolved` cannont be found in temporary directory after `Swift Package Manager` installation.
```

Comment:
```suggestion
    /// Thrown when `Package.resolved` cannot be found in temporary directory after `Swift Package Manager` installation.
```

Reply:
I found similar typo in `CarthageInteractorError`. I fixed it as well.